src/chunkserver/chunkserver-common/cmr_disk.cc

```diff
@@ -173,17 +176,36 @@ void CmrDisk::open(IChunk *chunk) {
 }
 
 int CmrDisk::unlinkChunk(IChunk *chunk) {
-	int result = 0;
+	// Get absolute paths for meta and data files
+	const std::filesystem::path metaFile = chunk->metaFilename();
+	const std::filesystem::path dataFile = chunk->dataFilename();
+
+	// Use the metaPath() and dataPath() to get the disk paths
+	const std::string metaDiskPath = FDDisk::metaPath();
+	const std::string dataDiskPath = FDDisk::dataPath();
```

Comment:
Consider using direclty metaPath() and dataPath(). Calling it with the FDDisk looks gives the impression of static and it is not needed in this context.